transport/src/main/java/io/camunda/zeebe/transport/TransportFactory.java

```diff
@@ -32,4 +41,23 @@ public ClientTransport createClientTransport(final MessagingService messagingSer
     actorSchedulingService.submitActor(atomixClientTransportAdapter);
     return atomixClientTransportAdapter;
   }
+
+  public <M extends BufferReader, P extends BufferWriter>
```

Comment:
:+1: I already ran into issues with the metadata marker, let's just use `BufferReader` directly for now :smile: